extension/src/popup/components/manageAssets/SearchAsset/index.tsx

```diff
@@ -99,7 +99,7 @@ export const SearchAsset = ({ setErrorAsset }: SearchAssetProps) => {
       try {
         res = await fetch(
           `https://api.stellar.expert/explorer/${
-            isTestnet ? "testnet" : "public"
+            isNetworkTestnet ? "testnet" : "public"
```

Comment:
Open question that needs to be addressed in a future PR: what stellar.expert API should we use for people on a custom network?